students/Ben_freeman/lesson09/pngdiscover.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+
+big_list = []
+
+
+def dive(directory):
```

Comment:
Pass _big_list_ as a parameter. Because of how lists work in Python, you will not be passing a copy of big_list, but a pointer to it (i.e., your function will be appending file names to the original list):

`def dive(directory, big_list)`